application/src/main/kotlin/application/ExamplesCommand.kt

```diff
@@ -174,12 +176,25 @@ For example:
         )
         var filterNot: List<String> = emptyList()
 
-        @Option(names = ["--contract-file"], description = ["Contract file path"], required = true)
-        lateinit var contractFile: File
+        @Option(names = ["--contract-file", "--spec-file"], description = ["Contract file path"], required = false)
+        var contractFile: File? = null
 
         @Option(names = ["--example-file"], description = ["Example file path"], required = false)
         val exampleFile: File? = null
 
+        @Option(names = ["--examples-dir"], description = ["Examples directory path associated to a single spec"], required = false)
```

Comment:
External examples directory path for a single API specification (If you are not following the default naming convention for external examples directory)

Reply:
Addressed